t/core/lrucache.t

```diff
@@ -204,3 +204,40 @@ release: {"_cache_ver":"t1","name":"aaa"}
 obj: {"_cache_ver":"t2","name":"bbb"}
 --- no_error_log
 [error]
+
+
+
+=== TEST 6: invalid_stale = true
```

Comment:
Is this test case for DNS?

Reply:
Yes, the DNS cache requires the parameter `invalid_stale = true`. It cannot reuse stale data first.